tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -320,10 +320,16 @@ When('the user unmarks the favorited file/folder {string} using the webUI', func
   return client.page.FilesPageElement.filesList().unmarkFavorite(path)
 })
 
-Then(/there should be no files\/folders listed on the webUI/, function () {
-  return client.page.FilesPageElement.filesList().allFileRows(function (result) {
-    client.assert.equal(result.value.length, 0)
-  })
+Then('there should be no files/folders/resources listed on the webUI', async function () {
+  const emptyMessageVisible = await client
+    .page
+    .FilesPageElement.filesList()
+    .isEmptyMessageVisible()
+
+  const allRowsResult = await client.page.FilesPageElement.filesList().allFileRows()
+
+  assert.ok(emptyMessageVisible, 'Empty message must be visible')
```

Comment:
is the message empty, or is the message about the folder being empty? ;-)

Reply:
the message is about "folder is empty", see screen shot above.

I suspected that this wording will cause confusion, I'll adjust it

Reply:
renamed to "noContentMessage"